play/src/front/Chat/Connection/Proximity/ProximityChatRoom.ts

```diff
@@ -365,8 +368,14 @@ export class ProximityChatRoom implements ChatRoom {
         });
     }
 
-    public joinSpace(spaceName: string): void {
-        this._space = this.spaceRegistry.joinSpace(spaceName);
+    public joinSpace(spaceName: string, propertiesToSync: string[]): void {
+        this._space = this.spaceRegistry.joinSpace(spaceName, propertiesToSync);
+        const proximityChatRoomSimplePeer = this._space.getSimplePeer();
+        if (proximityChatRoomSimplePeer) {
+            // Set up manager of audio streams received by the scripting API (useful for bots)
+            this.scriptingOutputAudioStreamManager = new ScriptingOutputAudioStreamManager(proximityChatRoomSimplePeer);
```

Comment:
`ScriptingOutputAudioStreamManager` registers an answerer in iframeListener.
If we have 2 joinSpace calls, the second one will overwrite the previous one.
We should ensure we have only one ProximityChatRoom at a given time.

We can throw an Exception in registerAnswerer if an answerer already exists.